exporter/cat1_view_helper.go

```diff
@@ -83,6 +84,30 @@ func identifierForString(objs ...string) string {
 	return identifierFor([]byte(b))
 }
 
+// identifierForResultValue generates an MD5 representation of a ResultValue
+func identifierForResultValue(resultValue models.ResultValue) string {
+	var b []byte
+	codeSystemsInResultValue := make([]string, len(resultValue.Codes))
```

Comment:
Wouldn't you just initialize this the same way you do `codesInResultValue`? Because you only append later on, you don't need to run `make` with a starting length.